nlmod/mdims/mgrid.py

```diff
@@ -102,200 +101,74 @@ def get_cell2d_from_model_ds(ds):
     return cell2d
 
 
-def get_xy_mid_structured(extent, delr, delc, descending_y=True):
-    """Calculates the x and y coordinates of the cell centers of a structured
-    grid.
-
-    Parameters
-    ----------
-    extent : list, tuple or np.array
-        extent (xmin, xmax, ymin, ymax)
-    delr : int or float,
-        cell size along rows, equal to dx
-    delc : int or float,
-        cell size along columns, equal to dy
-    descending_y : bool, optional
-        if True the resulting ymid array is in descending order. This is the
-        default for MODFLOW models. default is True.
-
-    Returns
-    -------
-    x : np.array
-        x-coordinates of the cell centers shape(ncol)
-    y : np.array
-        y-coordinates of the cell centers shape(nrow)
+def refine(ds, model_ws=None, refinement_features=None, exe_name=None,
+           remove_nan_layers=True):
```

Comment:
Perhaps provide a little more info to the function name? refine_gridgen, refine_quadtree, refine_lgr?